1주차/3.웨비의 사진관/style.css

```
@@ -1,159 +1,265 @@
+html {
+  scroll-behavior: smooth;
+}
+
 body {
-    font-weight: bold;
+  font-weight: bold;
+  background-color: #ecffdc;
 }
 
 a {
-    text-decoration-line: none;
+  text-decoration-line: none;
 }
 
-a:hover { color:mediumaquamarine; text-decoration-line: underline; }
-
-.header1 {
-    display: flex;
-    align-items: center;
-    background-color: mediumaquamarine;
-    height: 40px;
+a:hover {
+  color: mediumaquamarine;
+  text-decoration-line: underline;
+}
 
+ul {
+  display: flex;
+  justify-content: center;
+  padding-left: 0;
+  list-style: none;
 }
 
-.header {
-    display: flex;
-    align-items: center;
-    background-color: mediumaquamarine;
-    height: 40px;
-    position: sticky;
-    top: 0; /* 화면 상단에 고정 */
+li {
+  padding: 0 0.5rem;
+}
 
+.main-title {
+  display: flex;
+  align-items: center;
+  padding-left: 1rem;
+  background-color: mediumaquamarine;
+  height: 3rem;
 }
 
+.country {
+  display: flex;
+  align-items: center;
+  padding-left: 1rem;
+  background-color: mediumaquamarine;
+  height: 3rem;
+  position: sticky;
+  top: 0; /* 화면 상단에 고정 */
+}
 
 .img-preview {
-    display: flex;
-    overflow-x: auto; /*가로 스크롤바*/
-    height: 450px;
-    align-items: center;
+  display: flex;
+  overflow-x: auto; /*가로 스크롤바*/
+  margin: 0 1rem;
+  height: 28rem;
+  align-items: center;
+  position: relative;
+}
+
+::-webkit-scrollbar {
+  width: 1rem;
+}
+
+.img-preview::-webkit-scrollbar-thumb {
+  background-color: mediumaquamarine;
+  border-radius: 3rem;
 }
 
 .img-preview img {
-    width: 300px;
-    height: 400px;
-    margin-left: 5px;
-    margin-right: 5px;
+  width: 20rem;
+  height: 25rem;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
 }
 
 .nav {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-    height: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  height: 7rem;
 }
 
 .img-france {
-    display: flex;
-    flex-wrap: wrap;
-    height: 450px;
-    align-items: center;
-    justify-content: space-around;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 1rem 3.5rem;
+  gap: 1rem;
+  height: auto;
+  align-items: center;
+  flex-grow: 1;
+  justify-content: center;
+}
+.each {
+  position: relative; /*필요 */
+  width: 14rem;
+  height: 14rem;
+  flex-grow: 1;
+  text-align: center;
+  object-fit: cover;
 }
 
 .img-france img {
-    width: 17%;
-    height: 47%;
-    margin-left: 5px;
-    margin-right: 5px;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
 }
 
-.img-portugal {
-    display: flex;
-    height: 450px;
-    margin: 0 auto;
-    justify-content: center;
+.hover-text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 100%;
+  transform: translate(-50%, -50%); /* 텍스트를 가운데로 이동 */
+  color: #ffffff;
+}
+
+.hover-text p {
+  overflow: hidden;
+  white-space: normal;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```

과제 조건에 3줄 이상인 설명에 말 줄임표를 넣으라고 되어있는데, 다현이 코드에는 2줄 제한으로 되어있네..!!
```suggestion
  -webkit-line-clamp: 3;
```

요렇게 수정해줘야 될 것 같다 !

---

허거덩 그렇네 고마워